path_login.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/hashicorp/vault/sdk/logical"
 )
 
+var resourceClientAPIVersion = "2022-03-01"
```

Comment:
Might be worth dropping a comment on here with a link to where these versions can be found? Thinking if we need to update it in the future.

Reply:
Unfortunately this is not documented at all except to say that it is the "API version to use". You can find this in the Azure Portal when looking at a resource JSON View. Interestingly the `sdk/resourcemanager/resources/armresources` module is the only one in the entire azure-sdk-for-go that allows setting this magic value. I am wondering if we could set it as empty string to always use the most recent. I just used this value since it is the most recent available in the Azure Portal resource JSON view.

Reply:
I think this is our best bet as far as documentation goes: https://learn.microsoft.com/en-us/rest/api/resources/resources/get-by-id#uri-parameters

I went ahead and changed it to the value defined there and tested that everything works as expected

Reply:
Ah, gotcha. Thanks for the info and for adding the link in the comment. Looks good to me.